diff --git a/dist/runtime/native/styles.js b/dist/runtime/native/styles.js
index 533c93e10472247730d7fba96bfbf58333f4c49e..fa0465d09f16f23d8bd6580fe857c8a26ec83c7d 100644
--- a/dist/runtime/native/styles.js
+++ b/dist/runtime/native/styles.js
@@ -95,8 +95,7 @@ function injectData(data) {
         initiateStyle(style);
     }
     if (data.keyframes) {
-        for (const entry of data.keyframes) {
-            const value = keyframes.get(entry[0]);
+        for (const [entry, value] of Object.entries(data.keyframes)) {
             if (value) {
                 value.set(entry[1]);
             }